build.gradle.kts

```diff
@@ -13,3 +13,21 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
+plugins {
+  `java-library`
+  `maven-publish`
+}
+
+repositories {
+  mavenLocal()
+  maven { url = uri("https://repo.maven.apache.org/maven2/") }
```

Comment:
is this repo for special apache iceberg artifacts that we need?

or could we just use `mavenCentral()` here instead?

Reply:
I have followed the style from CEL java or Nessie. 

I will change it to mavenCentral()